Sources/Hummingbird/Router/Parameters.swift

```diff
@@ -46,7 +46,7 @@ public extension Parameters {
     /// - Parameter s: parameter id
     func require(_ s: String) throws -> String {
         guard let param = self[s[...]].map({ String($0) }) else {
-            throw HTTPError(.badRequest)
+            throw HTTPError(.badRequest, message: "Expected parameter '\(s)' does not exist")
```

Comment:
Likewise here, we should consider not exposing too much info

Reply:
Removed parameter name and replaced with text